### ISSUE TITLE
GPU-Z Json Output Error

### ISSUE THREAD

**Issue by Alliance-Ryan** [Demion/remotehwinfo#11]
When using GPU-Z I get a null value populated with "-nan(ind)" rather than null or 0. Not sure if this a big deal for other scripting languages but when querying the results with powershell it will output the broken json as text rather than a json

```json
{
        "entryIndex": 85,
        "name": "Crossbar Clock",
        "unit": "MHz",
        "digits": 1,
        "value": -nan(ind)
}
```
For anyone experiencing this issue a work around would be the following:
```powershell
$results.replace("-nan(ind)","0") | ConvertFrom-Json
```

**Comment by Demion** [comment]
Have you checked issue https://github.com/Demion/remotehwinfo/issues/9 ?

**Comment by Alliance-Ryan** [comment]
I didnt see that sorry, are you suggesting I use the dev build provided there?

**Comment by Demion** [comment]
Yes, try this dev build [RemoteHWInfo_v0.6a.zip](https://github.com/Demion/remotehwinfo/files/13804908/RemoteHWInfo_v0.6a.zip) or compile yourself latest from [master](https://github.com/Demion/remotehwinfo/tree/master).
